lms/templates/ccx/student_admin.html

```diff
@@ -2,10 +2,19 @@
 
 <section>
   <h2>${_('Student Grades')}</h2>
-  <p>
-    <a href="${gradebook_url}">${_('View gradebook')}</a>
-  </p>
-  <p>
+  <p class="ccx-grade-book-download">
+    <i class="fa fa-download" aria-hidden="true"></i>
     <a href="${grades_csv_url}">${_('Download student grades')}</a>
   </p>
+  <br/>
+  <div>
+    <iframe src='${gradebook_url}' id="ccx_grade_book_iframe"
+            name="ccx_grade_book_iframe"
+            frameBorder="0"
+            scrolling="no"
+            height="360px"
+            class="ccx-grade-book">
+      <p>Iframe not supported</p>
+    </iframe>
```

Comment:
@sarina any suggestion?
The goal of this PR was to minimize number of clicks to get grade book of ccx.
Now grade book has limit 20 students per page that why i embedded grade book here in iframe so that ccx coach can see his grade book in 2 one click from his dashboard.

Second pagination handling is easy in iframe, when user click next or previous page only iframe is refresh instead of whole student admin page.